src/openassetio-core/src/hostApi/Manager.cpp

```diff
@@ -46,9 +47,32 @@ ManagerPtr Manager::make(managerApi::ManagerInterfacePtr managerInterface,
       new Manager(std::move(managerInterface), std::move(hostSession)));
 }
 
+namespace {
+/**
+ * Extract the entity reference prefix from a manager plugin's info
+ * dictionary, if available.
+ */
+std::optional<Str> entityReferencePrefixFromInfo(const InfoDictionary &info) {
+  // Check if the info dict has the prefix key.
+  if (auto iter = info.find(Str{constants::kInfoKey_EntityReferencesMatchPrefix});
+      iter != info.end()) {
+    // Extract prefix string, and fail silently if the wrong type is
+    // stored under the key.
+    if (const auto *prefixPtr = std::get_if<openassetio::Str>(&iter->second)) {
+      return *prefixPtr;
+    }
+  }
+
+  // Prefix string not found, so return unset optional.
+  return {};
+}
+}  // namespace
+
 Manager::Manager(managerApi::ManagerInterfacePtr managerInterface,
                  managerApi::HostSessionPtr hostSession)
-    : managerInterface_{std::move(managerInterface)}, hostSession_{std::move(hostSession)} {}
+    : managerInterface_{std::move(managerInterface)},
+      hostSession_{std::move(hostSession)},
+      entityReferencePrefix_{entityReferencePrefixFromInfo(managerInterface_->info())} {}
```

Comment:
[fix] Sorry should have thought of this when we wrote it up - do you think we could log a `kDebugApi` message to state when the shortcut is activated and note that the managers own method wont be called?